Cargo.toml

```diff
@@ -149,6 +149,7 @@ async-graphql = { workspace = true, features = [
 ] }
 dotenvy = "0.15.7"
 convert_case = "0.6.0"
+oas3 = { git = "https://github.com/tailcallhq/oas3-rs", branch = "spotify" }
```

Comment:
Add a comment on exactly why are we using this fork. Create a PR to the main repository to get feedback about the changes.

Reply:
Use open-api 3.1 version like the maintainer has asked.